src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,79 @@
+import React from 'react';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const handlePrevClick = () => {
+    if (currentPage > 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const handleNextClick = () => {
+    if (currentPage < totalPages) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  return (
+    <>
+      <ul className="pagination">
+        <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={currentPage === 1}
+            onClick={handlePrevClick}
+          >
+            «
+          </a>
+        </li>
+        {[...Array(totalPages)].map((_, index) => {
+          const page = index + 1;
+
+          return (
+            <li
+              key={page}
+              className={`page-item ${page === currentPage ? 'active' : ''}`}
+            >
+              <a
+                data-cy="pageLink"
+                className="page-link"
+                href={`#${page}`}
+                onClick={() => onPageChange(page)}
+              >
+                {page}
+              </a>
+            </li>
+          );
+        })}
+        <li
+          className={`page-item ${currentPage === totalPages ? 'disabled' : ''}`}
+        >
+          <a
+            data-cy="nextLink"
+            className="page-link"
+            href="#next"
+            aria-disabled={currentPage === totalPages}
```

Comment:
The aria-disabled attribute is correctly set to true when the current page is the last page, ensuring accessibility compliance.